src/styles/components/_component-card.scss

```diff
@@ -1,20 +1,58 @@
-.component-link {
-  display: block;
-  outline: none;
+.component-card {
+  display: flex;
+  flex-direction: column-reverse;
+  height: 21.875rem; // 350px/16
   width: 100%;
-  height: 100%;
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  border: 2px solid var(--foreground-color);
+  background-color: var(--background-color);
 
-  .component-card {
-    display: flex;
-    flex-direction: column-reverse;
-    height: 21.875rem; // 350px/16
+  .component-link:focus & {
+    outline: none;
+    border: 2px solid var(--accent-color);
+  }
```

Comment:
This code was giving me double border! I think we can delete it because of the focus code you did on the link instead.